### ISSUE TITLE
[com.getpure.pure] Can't sign up with google

### ISSUE THREAD

**Issue by eternal-sorrow** [microg/GmsCore#2359]
**Affected app**
Name: [Pure]
Package id: [com.getpure.pure]

**Describe the bug**
When I try to signup, I press `Continue with Google` button, the app shows red message "Something went wrong" and nothing happens then.

**To Reproduce**
1. Launch the app
2. Wait until the startup animation ends
3. Tap the "Continue with Google" button

**Expected behavior**
Sign up in the app with Google.

**Screenshots**
I can add a screenshot if needed, but I thin I described the problem pretty well and it's unnecessary.

**System**
Android Version: 12
Custom ROM: lineageos4microg 19

**microG**
microG Core version: 0.2.27.223616
microG Self-Check results: All ticked

**Additional context**
none

**Comment by Wojtaz0w** [comment]
Your microG is outdated!

**Comment by eternal-sorrow** [comment]
There is nothing I can do about it.

**Comment by Wojtaz0w** [comment]
But you do know you can update it, right?

**Comment by Wojtaz0w** [comment]
I mean the microG app, is installing apk downloaded from GH releases impossible?

**Comment by eternal-sorrow** [comment]
I never download and install APKs manually. Only with f-droid. And it doesn't offer any microg updates for me.

**Comment by Wojtaz0w** [comment]
Supposedly, latest releases are marked as "unstable". ![Screenshot_20240506-135124_Neo Store.png](https://github.com/microg/GmsCore/assets/104674625/cbec8f55-1bcd-4700-a356-bc6c5d9dc67f)

**Comment by eternal-sorrow** [comment]
I wonder why...

**Comment by Wojtaz0w** [comment]
I do not know. On a side note, I'm always on the latest version and there are no "instabilities". It could however be due to Location Modules support being removed.

**Comment by eternal-sorrow** [comment]
I just updated it manually from f-droid. Now it works with Pure, but I instantly got a bug. The window "Allow this app to receive notifications" is very thin, like a tiny vertical strip in the middle of the screen and I had to scroll down to press "allow".

**Comment by Wojtaz0w** [comment]
> I just updated it manually from f-droid. Now it works with Pure, but I instantly got a bug. The window "Allow this app to receive notifications" is very thin, like a tiny vertical strip in the middle of the screen and I had to scroll down to press "allow".

Yes, that is known. I have reported that issue about a month ago and it is supposed to be fixed in next version, 0.3.2
